lib/rust/ensogl/core/src/debug/monitor.rs

```diff
@@ -823,10 +903,8 @@ mod tests {
         // Frame 1: simulate we managed to complete the work in 10ms, and then we wait 6ms before
         // starting next frame.
         //
-        // Note: there is no frame before "Frame 1", so the tested value for the previous frame
-        // will always be 0.0 and "Correct" at this point (which is a special case - depending on
-        // tested stat, for later frames 0.0 could result in a threshold warning/error).
-        test_and_advance_frame!(test, 0.0, ValueCheck::Correct; next: 10.0, 6.0);
+        // Note: there is no frame before "Frame 1", so the previous frame will not be tested.
```

Comment:
What do you mean by "there is no frame before "Frame 1""? Before the first rendering frame the program can evaluate expressions, so there is time before frame 1.

Reply:
Needs reworking of the comment and/or additional check for `None` value.